src/python/deepgnn/graph_engine/data/ppi.py

```diff
@@ -108,16 +118,16 @@ def _load_raw_graph(self, data_dir: str):
 
         return nodes, nodes_type, train_neighbors, other_neighbors, feats, class_map
 
-    def _build_graph(self, output_dir: str) -> str:
+    def _build_graph(self, output_dir: str, load_graph_output: tuple) -> str:
```

Comment:
why do we need to pass load_graph_output tuple instead of overriding it?

Reply:
If I call PPI's build_graph function from reddit, it will call PPI's load_graph function even if Reddit overloads load_graph. This is why I needed to refactor into constructor.

Reply:
There is probably something else going on, because this code works perfectly fine:
![image](https://user-images.githubusercontent.com/134559/208496701-c1f938fc-b104-41d1-baa7-c18a014ffcd6.png)